.github/workflows/unstable.yml

```diff
@@ -0,0 +1,33 @@
+name: unstable
+
+on:
+  push:
+    branches:
+      - master
+      - main
+    tags:
+      - ciflow/unstable/*
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref_name }}-${{ github.ref_type == 'branch' && github.sha }}-${{ github.event_name == 'workflow_dispatch' }}
+  cancel-in-progress: true
+
+jobs:
+  # There must be at least one job here to satisfy GitHub action workflow syntax
+  introduction:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    steps:
+      - name: Introduce PyTorch unstable workflow
+        run: |
+          echo "PyTorch unstable workflow is used to host experimental or flaky jobs"
+          echo " that needs to be run for every commit, but doesn't block PR merging"
+          echo " as part of the stable pull or trunk workflows."
+          echo
+          echo "In addition, a new label called ciflow/unstable can be attached to the"
+          echo " PR to trigger this workflow. That can be done either manually or"
+          echo " automatically using PyTorch auto-label bot."
+          echo
+          echo "Once the jobs are deemed stable enough (% red signal < 20% and TTS < 3h),"
```

Comment:
Most (~88% IIRC) of our workflows have %red signal < 5%. Let's set that as the target benchmark

Reply:
As this one is green, I think I will merge this one as it is and update the value to 5% later (in my PR to move ROCm here)